### ISSUE TITLE
Docker FPM example Compose File gives an "oci runtime error"

### ISSUE THREAD

**Issue by robclay** [nextcloud/docker#207]
Using the Docker Compose "Base version - FPM" example from [https://hub.docker.com/r/_/nextcloud/](https://hub.docker.com/r/_/nextcloud/) on a QNAP device:

I get 
"Cannot start service web: rpc error: code = 2 desc = "oci runtime error: could not synchronise with container process: not a directory"

It is due to the following line in the nginx setup: 
 volumes:
      - ./nginx.conf:/etc/nginx/nginx.conf:ro

Is there a different way to do this?

**Comment by schliflo** [comment]
You must not mount your nginx configuration. Just copy it in build stage. Something along the lines like here should get you started: https://github.com/schliflo/bedrock-docker/blob/master/.docker/web/Dockerfile
See https://github.com/nextcloud/docker/issues/104 as well ;)

**Comment by robclay** [comment]
Using Issue #104 as a go-by.

I have the docker compose file in a directory "tctest" - I created a directory within tctest called nginx:
qnap box:
/share
/share/tctest (Docker Compose)
/share/tctest/nginx (Dockerfile Here)

In my Dockerfile (Named "Dockerfile"):
`FROM nginx`
`COPY nginx.conf /etc/nginx/nginx.conf`

From my Dokcer compose file, I removed 
```
    volumes:
      - ./nginx.conf:/etc/nginx/nginx.conf:ro
```
Which runs now... Thank you... :) 

I go to my install: Firefox: IP.Address:8090 (Qnap uses 8080) and I get:

"Welcome to nginx!
If you see this page, the nginx web server is successfully installed and working. Further configuration is required..."

Do I need to edit the nginc.conf?  And if so, whee would it be?

Thanks for any help...
